.env-testnet-sample

```diff
@@ -62,3 +69,11 @@ LTC_RPC_PASS=sparkswapltc
 #
 EXTERNAL_BTC_ADDRESS=sample.ip.address
 EXTERNAL_LTC_ADDRESS=sample.ip.address
+
+# Docker compose files
+# docker-compose by default will look for the docker-compose.yml file and apply the docker-compose.override.yml file if
+# it exists by default. If we specify the COMPOSE_FILE variable, all compose files joined by the COMPOSE_PATH_SEPARATOR will
+# be applied and used
+#
+COMPOSE_PATH_SEPARATOR=:
```

Comment:
Is there a reason that `:` isn't the path separator on windows? Will this work on windows?

Reply:
https://github.com/docker/compose/pull/4588, COMPOSE_PATH_SEPARATOR was added so that it uses this as the separator no matter what.